react/__tests__/ProductSummaryImage.test.js

```diff
@@ -43,9 +44,11 @@ describe('<ProductImage /> component', () => {
     const product = ProductSummary.mapCatalogProductToProductSummary(rawProduct)
 
     const { container } = render(
-      <ProductSummary product={product}>
-        <ProductImage />
-      </ProductSummary>
+      <MockedProvider>
```

Comment:
vtex test-tools already add a MockedProvider:
https://github.com/vtex/test-tools/blob/master/src/react.tsx#L86

Do you really need this?

Reply:
This project was using an old vtex/test-tools version, probably this one doesn't have the MockedProvider. I updated it and now It's working. I removed all `<MockedProvider>` from the code.